README.md

```diff
@@ -1,24 +1,65 @@
-# README
+# DRAI
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is the production code for a tool designed for Assisters/Supervisors of community based organizations(CBOs), and will not be directly accessed by clients.
 
-Things you may want to cover:
+## Installation
 
-* Ruby version
+### Setup
 
-* System dependencies
+Run [`bin/setup`](bin/setup).
```

Comment:
This makes me wonder if we should have the Seeder be self-documenting so that it outputs the credentials for the various seed users. e.g. having these lines in the Seeder also include something like:

```ruby
puts "==== Admin User====\n"
puts "Email: admin@codeforamerica.org"
puts "Password: Qwerty!2"
```
etc. so that when someone runs the setup script they get the info they need to log in. 

Otherwise we should document in the readme that they should look in the Seeder.rb file

Reply:
We should also make sure to say to run `bin/rails s`. I always struggle to know how much to explain that this is a boring vanilla Rails app that should be usable by anyone who does Rails development, and how much to get into nitty-gritty setup details. But probably just referencing how to start the webserver is enough.